### PR TITLE
Trend Micro C2C docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create C2C docs for Trend Micro.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[C2C Trend Micro Vision One - Update Public Doc](https://sumologic.atlassian.net/browse/DOCS-557)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+   | United Arab Emirates | `api.mea.xdr.trendmicro.com` |
+   | United States | `api.xdr.trendmicro.com` |
+   | United States (for Government) | `api.usgov.xdr.trendmicro.com` |
+- **Bearer token**. To generate the authentication token, refer to the [Trend Micro Documentation](https://docs.trendmicro.com/en-us/documentation/article/cloud-app-security-integration-api-online-help-generating-an-authen).
+
+### Source configuration
+
+When you create an Trend Micro source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure an Trend Micro source:
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:53]:
```suggestion
To configure a Trend Micro source:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+1. **API Base URL**. Enter the [Base URL](#vendor-configuration) to fetch the data from the Trend Micro Vision One source.
+1. **Auth Token**. Enter the authentication token collected from the [Trend Micro platform](#vendor-configuration).
+1. **Polling Interval**. The polling interval is set for 15 minutes by default. You can adjust it based on your needs. This sets how often the Source checks for new data.
+1. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [SON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:70]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+   | Japan | `api.xdr.trendmicro.co.jp` |
+   | Singapore | `api.sg.xdr.trendmicro.com` |
+   | United Arab Emirates | `api.mea.xdr.trendmicro.com` |
+   | United States | `api.xdr.trendmicro.com` |
+   | United States (for Government) | `api.usgov.xdr.trendmicro.com` |
+- **Bearer token**. To generate the authentication token, refer to the [Trend Micro Documentation](https://docs.trendmicro.com/en-us/documentation/article/cloud-app-security-integration-api-online-help-generating-an-authen).
+
+### Source configuration
+
+When you create an Trend Micro source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:51]:
```suggestion
When you create an Trend Micro source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+To configure an Trend Micro source:
+1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. 
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **Trend Micro**.
+1. Enter a **Name** for the Source. The description is optional.
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:58]:
```suggestion
1. (Optional) For **Source Category**, enter any string to tag the output collected from the source. Category metadata is stored in a searchable field called `_sourceCategory`.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **API Base URL**. Enter the [Base URL](#vendor-configuration) to fetch the data from the Trend Micro Vision One source.
+1. **Auth Token**. Enter the authentication token collected from the [Trend Micro platform](#vendor-configuration).
+1. **Polling Interval**. The polling interval is set for 15 minutes by default. You can adjust it based on your needs. This sets how often the Source checks for new data.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:64]:
```suggestion
1. **Polling Interval**. The polling interval is set for 15 minutes by default. You can adjust it based on your needs. This sets how often the source checks for new data.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+1. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Save**.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:66]:
```suggestion
1. When you are finished configuring the source, click **Save**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+| config | JSON Object | [Configuration object](#configuration-object) | Yes | Source type specific values. |
+
+### Configuration Object
+
+| Parameter | Type | Required | Default | Description | Example |
+|:--|:--|:--|:--|:--|:--|
+| name | String | Yes | `null` | Type a desired name of the source. The name must be unique per Collector. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_source`. | `"mySource"` |
+| description | String | No | `null` | Type a description of the source. | `"Testing source"`
+| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
+| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field _siemForward to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:85]:
```suggestion
| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or source. Use the boolean field _siemForward to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
```

**Review by jpipkin1 — APPROVED** [review]
@JV0812, I have minor suggestions. I also added the logo (see commit "[Add logo image](https://github.com/SumoLogic/sumologic-documentation/pull/4776/commits/2067f086788070a3a1d7dbadde7949cd3e2f10d2)").

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 8]
```diff
@@ -481,6 +481,12 @@ In this section, we'll introduce the following concepts:
         <p>Learn how to collect event logs using the Trellix mVision ePO.</p>
         </div>
       </div>
+      <div className="box smallbox card">
+        <div className="container">
+        <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source"><img src={useBaseUrl('img/send-data/webex-logo.png')} alt="Thumbnail icon" width="100"/><h4>Trend Micro</h4></a>
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:486]:
```suggestion
        <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source"><img src={useBaseUrl('img/send-data/trend-micro-logo.png')} alt="Thumbnail icon" width="150"/><h4>Trend Micro</h4></a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+  - trend-micro
+description: Learn how to collect alert details from Trend Micro platform.
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/trend-micro/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/trend-micro/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/trend-micro/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/trend-micro-logo.png')} alt="webex-logo" width="120" />
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trend-micro-source.md:16]:
```suggestion
<img src={useBaseUrl('img/send-data/trend-micro-logo.png')} alt="logo" width="120" />
```

**Pull request merged**